src/app/templates/_.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
```

Comment:
Aren't all new projects on Travis on the container-based infrastructure by default?